src/components/Avatar/Avatar.scss

```diff
@@ -0,0 +1,161 @@
+@import "../../styles/themes.scss";
+@import "../../styles/typography.scss";
+@import "../../styles/states.scss";
+
+.monday-style-avatar {
+  position: relative;
+  &_circle {
+    position: relative;
+    border: 1px solid;
+    border-radius: 50%;
+    overflow: hidden;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    @include theme-prop(color, text-color-on-primary);
+
+    &:focus {
```

Comment:
focus happens on click - you can use :focus-visible, lets talk about what we are doing with it